README.md

```diff
@@ -19,24 +19,18 @@
   <br />
 </div>
 
-<img alt="Urkel" src="https://images-production.global.ssl.fastly.net/uploads/posts/image/97733/jaleel-white-steve-urkel.jpg" />
+<img width="965" alt="screen shot 2019-02-18 at 15 07 10" src="https://user-images.githubusercontent.com/1457604/52959744-ee6cef80-338e-11e9-96fe-cf5231b8eab7.png">
 
 ## ✨ Features
 
-- **One package** to get a working GraphQL client in React
-- Fully **customisable** behaviour via "exchanges"
-- Sane but simple default behaviour and document caching
-- Minimal React components and hooks
+- 📦 **One package** to get a working GraphQL client in React
+- ⚙️ Fully **customisable** behaviour via "exchanges"
+- 🗂 Sane but simple default behaviour and document caching
+- ⚛️ Minimal React components and hooks
 
-`urql` is mainly a GraphQL client, but also exposes a set of React components and hooks.
-It's built to be highly customisable and versatile, so you can take it from building
-your first GraphQL app, to building full and comples apps, or all the way to experimenting
-with GraphQL clients.
+This is a GraphQL client that exposes a set of React components and hooks. It's built to be highly customisable and versatile so you can take it from getting started with your first GraphQL project all the way to building complex apps and experimenting with GraphQL clients.
```

Comment:
> `urql` is a GraphQL client [...]

It's good to explicitly start the sentence with `urql` instead of `this` since no actual prose before has mentioned it before this paragraph :)